airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_metrics.json

```diff
@@ -131,8 +131,7 @@
       "type": ["null", "string"]
     },
     "initially_assigned_at": {
-      "type": ["null", "string"],
-      "format": "date-time"
+      "type": ["null", "string"]
```

Comment:
not date-time anymore?

Reply:
there was a CI bug with optional datetime fields. But it was fixed. I've restored as was